stdlib/REPL/src/REPL.jl

```diff
@@ -1284,28 +1284,28 @@ let matchend = Dict("\"" => r"\"", "\"\"\"" => r"\"\"\"", "'" => r"'",
             i = findnext(r"\"(?!\"\")|\"\"\"|'|`(?!``)|```|#(?!=)|#=", code, pos)
             isnothing(i) && break
             match = SubString(code, i)
-            j = findnext(matchend[match]::Regex, code, last(i) + 1)
+            j = findnext(matchend[match]::Regex, code, nextind(code, last(i)))
```

Comment:
This commit is not necessary, because the match string `code[i]` always consists of ASCII characters, and `code` is always UTF-8.

Reply:
I see! I made the change because of this test failure: https://build.julialang.org/#/builders/54/builds/594/steps/4/logs/stdio 

which is pointing to a `findprev` call, so I just (blindly) changed all `+ 1` and `- 1` operations.